e2e/_suites/fleet/fleet.go

```diff
@@ -1127,6 +1127,80 @@ func (fts *FleetTestSuite) upgradeAgent(version string) error {
 	return nil
 }
 
+func (fts *FleetTestSuite) checkDataStream() error {
```

Comment:
do we think it is worthwhile to abstract this a bit now, or wait for later?  I am hopeful we'll have more datastream tests, so at a minimum we might rename it to be specific to Linux if not any more abstracted.

If we do abstract it, it gets a little messier now perhaps, but the re-use could speed up other test additions, if we think the query is generic enough applied.